### ISSUE TITLE
could replace response or redirect request support directory redirection or replacement?

### ISSUE THREAD

**Issue by lovetingyuan** [requestly/requestly#477]
<img width="866" alt="image" src="https://user-images.githubusercontent.com/7310471/229742052-fabfb5d3-4c16-4038-a0f0-16c454ca38a1.png">
 
I want to use the directory "/user/document/app/static/scripts/" to replace any request including "/static/scripts/*.js".
The directory "/user/document/app/static/scripts/" contians a lot of javascript files, my website will request some js, eg "/static/scripts/a.js", "/static/scripts/b.js"..., now I have to set the rules one by one, I want to replace those requests using the whole directory.

**Comment by ashishsangle707** [comment]
Hey @lovetingyuan, 

Thanks for opening the ticket. This is Ashish from the Product team. Like you mentioned, Right now we only support redirecting to individual files.

This is a good feature request. This is not difficult to implement, we will discuss this internally and pick this up next week.

**Comment by sachinjain024** [comment]
@jasonreiche Do you also use "Map Local" feature and need a feature like this to redirect at directory level?

**Comment by sachinjain024** [comment]
@lovetingyuan One possible option that we are considering to support directory-level redirects is to use WildCard Operator and use group values in the destination file path something like this

```
URL "Matches (WildCard)" */static/scripts/*.js
Redirect To file:///user/document/app/static/scripts/$2.js
```

Does this work for you?

**Comment by jasonreiche** [comment]
Yes, this does sound like it could work to address the need, as long as the wildcard can include folders.  Our JS and CSS are in dozens of sub folders.

**Comment by sachinjain024** [comment]
@jasonreiche Just to clarify, Wildcard can be used in the URL match pattern and you can substitute the group values in the destination fields. 

Could you please help with an example of Redirect you want to set up using Wildcards?

**Comment by lovetingyuan** [comment]
> 

yes, looks good to me, thank you

**Comment by jasonreiche** [comment]
> @jasonreiche Just to clarify, Wildcard can be used in the URL match pattern and you can substitute the group values in the destination fields.
> 
> Could you please help with an example of Redirect you want to set up using Wildcards?

Example redirect rule: host.com/libs/* -> c:/libs/$1

Example redirects:
host.com/libs/abc/1.2.3/abc.js -> c:/libs/abc/1.2.3/abc.js
host.com/libs/abc/1.2.3/abc.css -> c:/libs/abc/1.2.3/abc.css
host.com/libs/def/1.0.1/def.js -> c:/libs/def/1.0.1/def.js
host.com/libs/ghi/7.0.0/ghi.js -> c:/libs/ghi/7.0.0/ghi.js

**Comment by sachinjain024** [comment]
Thanks @jasonreiche for the clarification. I think we have enough information now to start work on this.

**Comment by Ronakkadhi** [comment]
Closing as stale — no activity for over a year.

As part of the 2026 repository restructure, we're closing dormant issues to focus on actively-tracked items. If this is still relevant, please re-open or file a fresh issue at the right home:

- 🌐 **HTTP Interceptor** (Chrome extension + web app + desktop proxy) → [requestly/interceptor](https://github.com/requestly/interceptor/issues/new/choose)
- 🚀 **Requestly API Client** → this repo

Thanks for taking the time to file the original report.
